src/main/java/io/arivera/oss/embedded/rabbitmq/EmbeddedRabbitMqConfig.java

```diff
@@ -446,9 +456,14 @@ public EmbeddedRabbitMqConfig build() {
           erlangCheckTimeoutInMillis,
           cacheDownload, deleteCachedFile,
           envVars,
-          processExecutorFactory);
+          processExecutorFactory,
+          downloadProxy);
     }
 
+    public Builder downloadProxy( String hostname, int port ) {
```

Comment:
I think it's best that you can do here is to take the Proxy as an argument, instead of the hostname+port; otherwise, people who need another kind of Proxy (SOCKS for example) can't benefit from this feature.

You could potentially _add_ an overloaded method that takes the proxy and keep this method as is, but tbh, building a Proxy is so simple that this "helper" feels a bit unnecessary.